### PR TITLE
Created Sign-in page.

### PR DESCRIPTION
**Pull request by TDimuth**

### CODE REVIEW

File: dear-diary/src/containers/SignInPage/SignInCard.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        } else {
+            setIsNicknameValid(true);
+        }
+    };
+
+    const handleContinue = () => {
+        console.log({
+            name: nickname,
+        });
+        navigate('/Diary');
```
Comment (shamal-iroshan) [inline dear-diary/src/containers/SignInPage/SignInCard.tsx]: better to store these routes as a constants and use that constant to navigate

File: dear-diary/src/containers/SignInPage/SignInCard.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    };
+
+    const createRandomName = () => {
```
Comment (shamal-iroshan) [inline dear-diary/src/containers/SignInPage/SignInCard.tsx]: can move these kind of functions to util file

**Review by shamal-iroshan — CHANGES_REQUESTED** [review]
fix commented things,
When managing folder structure move components to a separate directory
ex:- SignInCard.tsx file should move to components/SignInCard/SignInCard.tsx

**Comment by TDimuth** [comment]
Fixed commented issues

**Review by shamal-iroshan — APPROVED** [review]

**Pull request merged**